hearthstone/entities.py

```diff
@@ -70,6 +71,8 @@ def register_entity(self, entity):
 		self.entities.append(entity)
 		if isinstance(entity, Player):
 			self.players.append(entity)
+		if not self.setup_done:
+			self.initial_entities.append(entity.id)
```

Comment:
I could be directly saving entity here, not sure what the performance/memory impact would be.